src/main/java/frc/robot/Cal.java

```diff
@@ -109,6 +109,17 @@ public static final class Lift {
         ELEVATOR_MAX_VELOCITY_IN_PER_SECOND = Cal.PLACEHOLDER_DOUBLE,
         ELEVATOR_MIN_OUTPUT_VELOCITY_IN_PER_SECOND = Cal.PLACEHOLDER_DOUBLE,
         ELEVATOR_ALLOWED_CLOSED_LOOP_ERROR_IN = Cal.PLACEHOLDER_DOUBLE;
+
+    /**
+     * Thresholds for when we consider the lift has reached a position. This is logical (for
+     * considering where the lift can go) but not functional (does not stop arm control or something
+     * when reached). We apply broader thresholds for the Starting position as the lift transits
+     * through this position.
+     */
+    public static final double ELEVATOR_THRESHOLD_INCHES = 0.5,
+        ARM_THRESHOLD_DEGREES = 2.0,
+        ELEVATOR_START_THRESHOLD_INCHES = 0.5,
+        ARM_START_THRESHOLD_DEGREES = 2.0;
```

Comment:
Either starting position thresholds should be larger, like the comment says, or that line in the comment should be removed. If we don't know how much larger they should be, make it `0.5 + PLACEHOLDER_DOUBLE`.

Reply:
Sure, I doubled them, we will almost certainly want to do more tuning later but this is a reasonable start